assets/js/rss_post_aggregation.js

```diff
@@ -407,7 +407,9 @@ window.RSS_Post_Aggregation = (function(window, document, $, undefined){
 
 	app.init = function() {
 		log( app );
```

Comment:
Wasn't aware this was here, going to make a second ticket to cleanup the JS after looking at this file.